cmd/runtime/main.go

```diff
@@ -26,10 +27,12 @@ import (
 	"github.com/cosi-project/runtime/pkg/state/protobuf/server"
 )
 
+var addressAndPort string
```

Comment:
Must it be a global variable?

Reply:
Was already the pattern used. I just went with it.

Reply:
Happy to address in a follow up, unless we feel it should be addressed in this PR.